justfile

```diff
@@ -76,9 +76,16 @@ qemu: build-linux nixos-image
     -m 512M \
     -cpu host \
     -virtfs local,path={{invocation_directory()}}/..,security_model=none,mount_tag=home \
+    -virtfs local,path=/scratch,security_model=none,mount_tag=scratch \
     -nographic -enable-kvm \
     -s
 
+vm_mounts:
```

Comment:
I can also make this of the nixos configuration if you want.

Reply:
i'd be glad if you could do that

Reply:
I add the mounts later on.